templates/web.ipv6.template.yml

```diff
@@ -1,8 +1,2 @@
+# This file is deprecated; you can remove it from your app.yml
```

Comment:
Let's echo a deprecation message instead as it is highly unlikely that people will see this comment. Let's just set it for removal 1 year later?